InvenTree/stock/templates/stock/low_stock_notification.html

```diff
@@ -0,0 +1,27 @@
+<p>{{ message }}</p>
+
+<table style="border-collapse:collapse; width: 80%;margin-left: 10%; font-size: 1rem">
+
+    <tr style="background: aliceblue; height: 4rem;">
+        <th colspan="3" style="padding-bottom: 1rem; font-size: 1.5rem; color:rgb(210,0, 0)">Part low on stock</th>
+    </tr>
+
+    <tr style="height: 3rem; border-bottom: 1px solid">
+        <th>Part Name</th>
+        <th>Available Quantity</th>
+        <th>Minimum Quantity</th>
+    </tr>
+
+    <tr style="height: 3rem">
+        <td style="text-align: center">{{ part_name }}</td>
+        <td style="text-align: center">{{ part_quantity }}</td>
+        <td style="text-align: center">{{ minimum_quantity }}</td>
+    </tr>
+
+    <tr style="background-color: aliceblue;height: 4rem;">
+        <td colspan="3" style="padding-top:1rem; text-align: center">You are receiving this mail because you have starred the part {{ part_name }} in
```

Comment:
These strings will need to be placed within a `{% trans "string" %}` block

@matmair in the case of sending an email, how does the server know which locale to encode in? With a web request this is included in the HTML headers, or we use the user specified locale...

Reply:
I changed the same in my latest commit based on the suggestions my @matmair .
Kindly refer the [commit](https://github.com/inventree/InvenTree/pull/2208/commits/6ec2801fcea15d0784e0ad57f6000e8b568aa05b)

Reply:
@SchrodingersGat it uses default locale(English), we currently have not implemented reports or messages with a correct language parsing; is on my todo after I get the base for plugins finished